Runar/Runar/Controllers/Misc/OnboardingScreen/OnboardingScreenCell.swift

```diff
@@ -62,9 +62,11 @@ class OnboardingScreenCell: UICollectionViewCell {
         contentView.addSubview(onboardingDescription)
         onboardingDescription.snp.makeConstraints { make in
             make.top.equalTo(onboardingSlideTitle.snp.bottom).offset(32)
-            make.width.equalTo(230)
             make.height.equalTo(70)
             make.centerX.equalTo(contentView)
+            let constant: CGFloat = DeviceType.iPhoneSE ? (viewWidth / 7.6) : (viewWidth / 5.45)
```

Comment:
@Kengii what does it mean 7.6 and 5.45?
It looks like Kalashnikov machine gun cartridge calibers.